outbox-core/src/main/java/io/praesidio/outbox/MessageRelay.java

```diff
@@ -17,26 +17,29 @@
 
     private final MessageRepository messageRepository;
     private final Map<MessageType, MessageRelayProvider> messageRelayProviders;
-    private final Map<MessageType, MessageSerializer> messageSerializers;
```

Comment:
:+1:  but I think it should be done in a separate commit ;)

Reply:
Seems there are different ideas where to handle serialization: https://github.com/praesidio/outbox/pull/31/files#diff-4a1f6738788525b16599e39e7404fa88951964bf404f28bdc70b143dde1c0480